Jenkinsfile

```diff
@@ -384,7 +384,7 @@ pipeline {
         stage('L2: Speech to Text RNNT WPE') {
           steps {
             sh 'STRICT_NUMBA_COMPAT_CHECK=false python examples/asr/speech_to_text_rnnt_bpe.py \
-            --config-path="experimental/contextnet_rnnt/" --config-name="config_rnnt_bpe.yaml" \
+            --config-path="conf/contextnet_rnnt/" --config-name="config_rnnt_bpe.yaml" \
```

Comment:
Are these Citrinet-RNNT or Contextnet?

Reply:
These are old small model configs just for CI. They wont train properly for LS. The new contextnet config is the actual trainable one.